microsoft-edge/devtools-guide-chromium/beginners/css.md

```diff
@@ -26,7 +26,7 @@ keywords: microsoft edge, web developement, f12 tools, devtools
 
```

Comment:
In the screenshot that's below "Choose New Style Rule (New Style Rule) which is next to .cls. An empty ruleset appears with a as the selector.", add red box around +.   eg: 
1. in VS Code, with doc tool extension installed, click the image link to display image in VS Code.  
2. shortcut Ctrl+Alt+R opens File Explorer w/ the file selected.
3. Right-click, Edit (in Paint).  
4. Zoom (Ctrl+PgUp).  3px wide red box. 
5. Save.
I jumped into that section w/o context, and it took a long time to figure out what the sentence was talking about in the screenshot.  Good example of why red boxes are required, for dense UI.